docsbox/Dockerfile

```diff
@@ -1,30 +1,40 @@
-RUN apt-get -y install -y --no-install-recommends software-properties-common dialog apt-utils \
-    libffi-dev libmagic-dev libmagickwand-dev libexempi8 exempi build-essential \ 
-    autoconf automake libtool ghostscript icc-profiles-free liblept5 libleptonica-dev libxml2 \
-    pngquant tesseract-ocr zlib1g zlib1g-dev ca-certificates curl git python3-dev python3-pip python3-cffi \ 
-    python3-pkg-resources python3-reportlab python3-distutils python3-pkg-resources python3-reportlab ocrmypdf
-RUN echo 'sudo deb-src http://ppa.launchpad.net/libreoffice/libreoffice-7-0/ubuntu focal main -y'
-RUN apt-get -y update
-RUN apt-get install -y libreoffice libreofficekit-dev hyphen-fi hyphen-fr hyphen-de hyphen-en-us hyphen-it hyphen-pl hyphen-ru
-RUN echo debconf debconf/frontend select Noninteractive | debconf-set-selections
-RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
-RUN apt-get install -y ttf-mscorefonts-installer
-RUN apt-get install -y texlive-fonts-extra fonts-dejavu-extra fonts-freefont-otf fonts-freefont-ttf fonts-lmodern fonts-lyx fonts-sil-gentium fonts-dustin fonts-fanwood fonts-3270 fonts-font-awesome fonts-bpg-georgian fonts-opensymbol
-RUN fc-cache -vr
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime \
+    && echo $TZ > /etc/timezone \ 
+    && apt-get -y update && apt-get install -y --no-install-recommends build-essential \
+    software-properties-common dialog apt-utils libffi-dev libmagic-dev libmagickwand-dev \
+    libexempi8 exempi autoconf automake libtool ghostscript icc-profiles-free \
+    liblept5 libleptonica-dev libxml2 pngquant tesseract-ocr zlib1g zlib1g-dev ocrmypdf git \
+    python3-dev python3-pip python3-cffi python3-reportlab python3-distutils python3-pkg-resources \
+    && rm -rf /var/lib/apt/lists/*
 
-RUN apt-get clean && apt-get -y update && apt-get install -y locales && locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US:en
-ENV LC_ALL en_US.UTF-8
+RUN echo 'sudo deb-src http://ppa.launchpad.net/libreoffice/libreoffice-7-0/ubuntu focal main -y' \
+    && apt-get -y update \
+    && apt-get install -y libreoffice libreofficekit-dev hyphen-fi \
+    hyphen-fr hyphen-de hyphen-en-us hyphen-it hyphen-pl hyphen-ru
+
+RUN echo debconf debconf/frontend select Noninteractive | debconf-set-selections \
+    && echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections \
+    && apt-get -y update \
+    && apt-get install -y --no-install-recommends \
+    ttf-mscorefonts-installer fonts-dejavu-extra fonts-freefont-otf fonts-freefont-ttf \ 
+    fonts-lmodern fonts-lyx fonts-sil-gentium fonts-dustin fonts-fanwood \
+    fonts-3270 fonts-font-awesome fonts-bpg-georgian fonts-opensymbol \
+    && fc-cache -vr && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get -y update \
+    && apt-get install -y locales \
+    && locale-gen en_US.UTF-8 \
+    && apt-get clean
```

Comment:
Would it be good to run the clean after every apt-get install above so the downloaded packages won't be added to the layer?